dashboard/app/assets/javascripts/admin/levelCompletions.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Shows the samplingMessage and completionTable DOM elements depending on the
+ * parameters.
+ *
+ * @param {boolean}: isSampled - whether Google Analytics sampled.
+ * @param {boolean} startDatePresent - whether a start_date parameter was given.
+ */
+//
```

Comment:
superminor nit: stray empty comment.

Also, `@param {boolean}: isSampled` above shouldn't have the `:`